README.md

```diff
@@ -109,6 +109,10 @@ BitTorrent) inside the web browser, as of today.
 
 See the [docker guide](/support/doc/docker.md)
 
+## Run on YunoHost
+
+Peertube app for [YunoHost](https://yunohost.org).See [here](https://github.com/YunoHost-Apps/peertube_ynh)
```

Comment:
There is a missing space before "See".

Also, why not add an "Install on with YunoHost" badge? https://github.com/YunoHost/install-app